packages/react/src/components/Masthead/__tests__/MastheadSearch.test.js

```diff
@@ -55,33 +40,30 @@ xdescribe('MastheadSearch', () => {
       },
     });
     input.simulate('focus');
-    await tick();
-
     expect(SearchTypeaheadAPI.getResults).toHaveBeenCalled();
   });
 
-  it('should redirect to the results page when a user clicks a suggestion', async () => {
-    const masthead = mount(<MastheadSearch />);
-    const input = masthead.find(
-      `[data-autoid="${stablePrefix}--header__search--input"]`
-    );
+  // it('should redirect to the results page when a user clicks a suggestion', async () => {
```

Comment:
@guilhermelMoraes These comments should be removed.

Reply:
okay, sure.